lib/debug.js

```diff
@@ -1,6 +1,6 @@
 var debug = require('debug');
 var name = require('../package.json').name;
-var MAX = 80;
+var MAX = 120;
```

Comment:
I'm going to make a strong-debug wrapper... this code is cut-n-paste too many places.